zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -7989,10 +7989,10 @@ components:
           description: The BPMN ID of the form.
```

Comment:
❓ Since we're at it right now: what is this information? The id I provide as a user in the form JSON and that I use in a user task to link to the form?

Reply:
Thanks for the question @tmetzke 

The ID refers to the Form ID present on the modeler, (similar how we do in tasks and process)
<img width="1242" alt="image" src="https://github.com/user-attachments/assets/635c3f9f-59f6-442d-bce7-2a7caf865104" />

We can be more specific over this field specifying what refers considering the modeler on the attribute description.
What do you think?

Reply:
OK, I see, thank you, @nathansandi 👍 
With this in mind, the name `bpmnId` doesn't make any sense for this property, if you ask me. It's actually the `formId` (we have to properly differentiate it from the `formKey` property then by description, e.g. `The user-provided identifier of the form` while the `formKey` rather is `The internal and unique system identifier of the form`).

Reply:
@pepopowitz, FYI, regarding the glossary idea, I believe it could make sense to provide a similar wording for all `id` versus `key` attributes in the API. IDs are usually user-provided identifiers then, whereas keys usually are system-generated (I think the only exception currently is the `correlationKey` used in messaging endpoints and in the future maybe `businessKey`).

Reply:
@nathansandi, can you create a follow-up to rename the `bpmnId` attribute here to `formId` and to adjust the description of this attribute and the `formKey`?

Reply:
Hello @tmetzke 
Follow up issue created:
https://github.com/camunda/camunda/issues/26911